app/views/content/life-as-a-teacher/teaching-as-a-career/abigails-career-progression-story/_article.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="row inset">
+  <section class="col col-720 col-space-l-top">
+      <h2>Becoming a maths teacher</h2>
+      <p>I always enjoyed maths at school and had the opportunity to take my maths GCSE early. I went on to study Maths and Management at university, always with the idea that I would go into a leadership or management role, but with no certainty of what industry I would enter. </p>
+      <p>A few years after I finished university, my old school sent out a letter to former students advertising their teacher training course. Around the same time, I went to meet one of my friends who was working at a [Get Into Teaching event](/events) and got chatting about a career in teaching. It seemed like a great opportunity!</p>
+      <p>I really enjoyed my teacher training. Although I had no experience in the classroom beforehand, I found I settled quite naturally into speaking to pupils at the front of the classroom. I also had a lot of support from my mentors and the maths team.</p>
+      <h2>Taking on a new role</h2>
+      <p>In the January of my early career teacher (ECT) year, my school advertised a numeracy coordinator role. The role was all about embedding numeracy into subjects across the whole school. Despite being an ECT, I was keen to progress and had a lot of ideas for the role, so I decided to go for it. I worked hard to prepare and I was so pleased to be offered the job!</p>
+      <p>The numeracy coordinator role was a new one in the school, so I could really make it my own. I decided to cement numeracy into form time, coming up with activities pupils could do to improve basic skills, and I worked with teachers in different departments to make sure we were being consistent in the way we were teaching maths. I also introduced a maths mastery homework competition across key stage 3. I had a lot of support and encouragement from my mentor and from teaching staff across the school.</p>
+      <h2>Increased responsibility</h2>
+      <p>In my third year I was promoted to second in department and took on more responsibility. I lead meetings, manage the budget, conduct appraisals, and organise ‘learning walks’ to ensure teaching is consistent across the department. This extra work is reflected in my teaching schedule, with reduced classroom hours so that I can manage all the additional tasks.</p>
```

Comment:
```suggestion
      <p>In my third year I was promoted to second in department and took on more responsibility. I lead meetings, manage the budget, conduct appraisals, and organise learning walks to ensure teaching is consistent across the department. This extra work is reflected in my teaching schedule, with reduced classroom hours so that I can manage all the additional tasks.</p>
```
Do we need to have the quotes here?